ppr-ui/src/components/common/Remarks.vue

```diff
@@ -22,12 +22,12 @@
               id="remarks-textarea"
               v-model.trim="remarks"
               filled
-              :rules="maxLength(420)"
-              name="name"
-              counter="420"
-              label="Remarks (Optional)"
+              :counter="remarksMaxLength"
+              :rules="maxLength(remarksMaxLength)"
+              :label="label"
               class="pl-1"
               data-test-id="remarks-textarea"
+              :error-messages="showRemarksRequiredMsg ? 'Enter remarks' : ''"
```

Comment:
For some reason, I couldn't get required error to trigger via `:rules` prop, so had to make this work-around.

Reply:
I think you need to return remarks form